tests/scripts/actionutils.sh

```diff
@@ -74,19 +74,31 @@ function enable_rgw() {
 }
 
 function create_containers() {
+    set -x
+    # Create public network for internal ceph cluster
+    lxc network create public
+    nw=$(lxc network list --format=csv | grep "public" | cut -d, -f4)
+    gw=$(echo $nw | cut -d/ -f1)
+    mask=$(echo $nw | cut -d/ -f2)
```

Comment:
Wonder if it's worthwhile to factor out the network param calculation into a function as it's repeated a few times through the script

Reply:
ack